hasweb/pages/site/tos.md

```diff
@@ -46,7 +50,7 @@ The terms of service specific to posting a listing are listed under the [Job Boa
 
 * Events
 
-HasGeek organizes events e.g. conferences, workshops, hacknights for the technology community, and incurs expenses for venue, food, snacks, stationery & printing, audio & video equipment as well as other rental or purchased equipment. HasGeek reserves entry to select events only to valid ticket holders. A ticket is valid only for the persons named on the ticket issued by us.
```

Comment:
Validity clause shouldn't be removed. Tickets are not transferable without authorization. Among other things, this is a legal requirement for liability in case of an incident -- we need to know who we are hosting.